src/hotspot/cpu/aarch64/aarch64_ad.m4

```diff
@@ -1,4 +1,4 @@
-dnl Copyright (c) 2014, Red Hat Inc. All rights reserved.
```

Comment:
Did you regenerate everything from .m4 files, or merge the generated sources from mainline? If not, please do and see if there's any difference.

Reply:
I don't remember, but I just checked by regenerating the `.ad` part via `m4` and there is no diff.